frontend/src/uxbox/main/refs.cljs

```diff
@@ -119,6 +119,14 @@
 (def selected-shapes
   (l/derived :selected workspace-local))
 
+(def selected-objects
+  (letfn [(selector [state]
+           (let [selected (get-in state [:workspace-local :selected])
+                 page-id (get-in state [:workspace-page :id])
+                 objects (get-in state [:workspace-data page-id :objects])]
+             (->> selected (map #(get objects %)))))]
```

Comment:
`mapv` instead of `map` should be used. `map` is lazy, then the `=` should be used for checking equality because the result of the selector is always a new list instance (that makes the default `identical?` checking useless).

Reply:
done